client/src/css/structure.css

```diff
@@ -0,0 +1,454 @@
+ * through definition
+ */
+.sidebar-group { 
+  border-top : 1px solid #ccc;
+  margin-top : 6px;
+  height : calc(100vh - 148px);
+  width : 100%;
+  overflow-y : auto;
+  overflow-x : hidden;
+}
+
+.expanded .link-label { 
+  padding-left : 5px;
+  display : inherit;
+}
+
+.link-label { 
+  white-space : nowrap; 
+  display : none;
+}
+
+/* Flex footer test */
+.flex-footer { 
+  font-size : 100%; 
+  overflow : hidden;
+  margin : 0;
+  padding : 0 0 0 0;
+  border-top : 1px solid #ccc;
+  position : absolute;
+  bottom : 0;
+  min-height : 40px;
+  width : 100%;
+}
+
+.flex-footer ul { 
+  margin : 0;
+  padding : 0;
+  display : flex;
+  flex-wrap : wrap;
+  justify-content : center;
+  list-style : none;
+}
+
+.flex-footer a { 
+  padding-top : 10px;
+  padding-bottom : 10px;
+  display : flex;
+  align-items : center;
+  padding-left : 20px;
+  padding-right : 20px;
+  color : #707070;
+  min-height : 48px;
+}
+
+.expanded .flex-footer li:first-child { 
+  flex-grow : 1; 
+}
+
+.flex-footer a:hover { 
+  background-color : #e6e6e6;
+  text-decoration : none;
+  color : #707070;
+  cursor : pointer;
+}
+
+.link-nav { 
+  width : 100%;
+  height : 100%;
+  display : flex;
+  justify-content : center; 
+  align-items : center;
+  border: none;
+  cursor : pointer;
+}
+
+.link-nav:hover {
+  background-color : #e6e6e6;
+}
+
+.expanded .link-nav { 
+  display : none;
+}
+
+.text-vertical { 
+  transform : rotate(-90deg);
+  color : #707070;
+}
+
+/** Hide the navigation link when sidebar is not expanded (default state) */
+.flex-nav { 
+  display : none;
+}
+
+.expanded .flex-nav { 
+  display : block;
+}
+
+/** 
+ * @todo Tree styles very temporarily defined - these should be carefully 
+ * refined 
+ */
+.flex-tree ul { 
+  margin : 0;
+  padding : 0;
+  display : flex;
+  list-style : none;
+  flex-direction : column;
+  width : 100%;
+}
+
+.flex-tree li { 
+  border-bottom : 1px solid #ccc;
+}
+
+.flex-tree a { 
+  display : flex;
+  align-items : center;
+  color : #707070;
+  min-height : 35px;
+  padding-left : 10px;
+}
+
+.flex-tree a:hover { 
+  background-color : #e6e6e6;
+  text-decoration : none;
+  color : #707070;
+  cursor : pointer;
+}
+
+.file { 
+  border-left : 5px solid #707070;
+}
+
+.tree-title { 
+  padding-left : 7px;
+}
+
+/** 
+ * Contains application modules (ng-view) - pushes margin right on sidebar 
+ * expansion
+ */
+.flex-view { 
+  margin-left : 56px;
+  display : flex;
+  -webkit-box-orient : vertical;
+  -webkit-box-direction : normal;
+  flex-direction : column;
+  overflow-x : hidden;
+  overflow-y : auto;
+  position : relative;
+}
+
+.expanded .flex-view { 
+  margin-left : 260px;
+}
+
+.selected a { 
+  color : #2089CE;
+  font-weight : bold;
+}
+.selected a:hover { 
+  color : #2089CE;
+}
+
+li.selected { 
+  border-left-color : #2089CE;
+}
+
+/** 
+ * Temporary header breadcrumb bootstrap fork
+ * @todo implement this in less using the boostrap element 
```

Comment:
Good idea, will do.

On Wednesday, January 27, 2016, Jonathan Niles <notifications@github.com>
wrote:

> In client/src/css/structure.css
> <https://github.com/IMA-WorldHealth/bhima-2.X/pull/42#discussion_r50952087>
> :
>
> > +
> > +.selected a {
> > +  color : #2089CE;
> > +  font-weight : bold;
> > +}
> > +.selected a:hover {
> > +  color : #2089CE;
> > +}
> > +
> > +li.selected {
> > +  border-left-color : #2089CE;
> > +}
> > +
> > +/**
> > + * Temporary header breadcrumb bootstrap fork
> > + * @todo implement this in less using the boostrap element
>
> Once this lands, you should make an issue about it so that we can keep
> track of these refinements.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/IMA-WorldHealth/bhima-2.X/pull/42/files#r50952087>.
>